module/x/gravity/handler_test.go

```diff
@@ -522,25 +522,21 @@ func TestMsgSendToCosmosClaimSpreadVotes(t *testing.T) {
 	require.NotNil(t, a3)
 	// and no additional added to the account
 	balance3 := input.BankKeeper.GetAllBalances(ctx, myCosmosAddr)
-	assert.Equal(t, sdk.Coins{sdk.NewInt64Coin("gravity0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e", 12)}, balance3)
+	assert.Equal(t, sdk.Coins{sdk.NewInt64Coin(denom, 12)}, balance3)
 }
 
 // Tests sending funds to a native account and to that same account with a foreign prefix
 // The SendToCosmosClaims should modify the balance of the underlying account
 func TestMsgSendToCosmosForeignPrefixedAddress(t *testing.T) {
 	var (
-		coreAddress          = "6ahjkfqxpp6lvfy9fpfnfjg39xr96qet"
-		myForeignBytes, err0 = types.IBCAddressFromBech32("levity1" + coreAddress + "vanuy5")
-		myForeignAddr        = sdk.AccAddress(myForeignBytes)
-		myNativeAddr, err1   = sdk.AccAddressFromBech32("gravity1" + coreAddress + "0l08hu")
-
-		myNonce      = uint64(1)
-		anyETHAddr   = "0xf9613b532673Cc223aBa451dFA8539B87e1F666D"
-		tokenETHAddr = "0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e"
-		myBlockTime  = time.Date(2020, 9, 14, 15, 20, 10, 0, time.UTC)
+		// the data needs to be 20 bytes of the same data, with there being different prefixes
+		myForeignAddr   = keeper.TestAccAddress("foreign", []byte("1___Test_Address___1"))
+		myNativeAddr    = keeper.TestAccAddress(types.GravityDenomPrefix, []byte("1___Test_Address___1"))
```

Comment:
Why it's '"1___Test_Address___1"' ?

Reply:
It needs to be 20 bytes of any data, but `myForeignAddr` and `myNativeAddr` need to have the same 20 bytes of data.